packages/framework/src/Resources/config/services.yml

```diff
@@ -146,6 +146,10 @@ services:
 
     Shopsys\FrameworkBundle\Component\Cron\Config\CronConfig: ~
 
+    Shopsys\FrameworkBundle\Component\Cron\CronModuleExecutor:
```

Comment:
What if I have overwritten this in project-base? Is it still OK?

Reply:
Yes, interface of class is still same